### PR TITLE
Handle relative import path for contracts in `test` command

### PR DESCRIPTION
**Pull request by m-Peter**
PR for context: https://github.com/SupunS/cadence-testing/pull/1

### CODE REVIEW

**Comment by m-Peter** [comment]
cc @SupunS @turbolent

**Comment by m-Peter** [comment]
Sure thing! I just wanted to see first, if anyone has a better implementation idea on how to handle this case.

**Comment by m-Peter** [comment]
@sideninja Added two test cases in https://github.com/onflow/flow-cli/pull/985/commits/1464fbec94253a45e4880a4f4e449891d725f992

**Review by turbolent — APPROVED** [review]
Nice!

**Review by SupunS — APPROVED** [review]
Thank you for fixing this!

File: pkg/flowkit/services/tests.go [hunk 1]
```diff
@@ -84,13 +84,17 @@ func (t *Tests) importResolver(scriptPath string, readerWriter flowkit.ReaderWri
...
-		if err != nil {
-			return "", err
+		relativePath := stringLocation.String()
+		for _, contract := range *t.state.Contracts() {
+			if !strings.Contains(relativePath, contract.Location) {
+				return "", fmt.Errorf(
+					"cannot find contract with location '%s' in configuration",
+					relativePath,
+				)
+			}
```
Comment (SupunS) [inline pkg/flowkit/services/tests.go:94]: Why does it need to terminate the loop on the first failure? wouldn't it need to search for all contracts?

Sorry, for the very late comment/question, just noticed this only now.

Comment (m-Peter) [reply pkg/flowkit/services/tests.go]: That's a great catch Supun :100: :100: 
I have fixed this in https://github.com/onflow/flow-cli/pull/985/commits/7dd7322c770da105aebed9dc133e57a0004ee7b3

**Review by SupunS — APPROVED** [review]

**Comment by m-Peter** [comment]
@sideninja Is there any plan for a release this week? If yes, we would like to have this bug-fix included, as it is a blocker towards adoption of the code coverage feature :pray:

**Comment by sideninja** [comment]
@m-Peter we can make a patch release today cc @chasefleming

**Comment by m-Peter** [comment]
If that's not too much trouble, it would be awesome :heart_eyes:

**Pull request merged**